plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -769,4 +780,18 @@ private void updateAllDependents(String parent, Plugin dependent, Map<String,Ver
         }
         return diff;
     }
+
+    private boolean checkJenkinsTestHarness(File buildLogFile, MavenPom pom) throws IOException {
+        try (FileInputStream is = new FileInputStream(buildLogFile)) {
+            String log = IOUtils.toString(is);
+
+            if (log.contains("Failure to find org.jenkins-ci.main:jenkins-test-harness")) {
+                pom.replaceDependencyVersion("org.jenkins-ci.main", "jenkins-test-harness", "RELEASE");
```

Comment:
I know using RELEASE is not always appropriate but since this is done in case there is already a failure, I don't think it's a bad thing to use the latest released version.

Anyway any suggestion is more than welcome!